src/tests/tablet/schedule/CannotCreateMeetingIfAttendeeIsInvalid.java

```diff
@@ -0,0 +1,46 @@
+package tests.tablet.schedule;
+
+import static framework.common.AppConfigConstants.EXCEL_INPUT_DATA;
+
+import java.util.List;
+import java.util.Map;
+
+import org.junit.Assert;
+import org.testng.annotations.Test;
+
+import framework.pages.tablet.HomeTabletPage;
+import framework.pages.tablet.SchedulePage;
+import framework.utils.readers.ExcelReader;
+
+/**
+ * TC34: Verify that Attendees text box doesn't allow a different format than email
+ * @author Asael Calizaya
+ *
+ */
+public class CannotCreateMeetingIfAttendeeIsInvalid {
+	SchedulePage schedule = new SchedulePage();
+	ExcelReader excelReader = new ExcelReader(EXCEL_INPUT_DATA);
+	List<Map<String, String>> meetingData = excelReader.getMapValues("MeetingData");
```

Comment:
Maybe you can use the method createdMeetingFromExcelFile (int row) that I have...you can review that....